### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mnumanuyar**
this project needs certain structs outside of the repo. This sometimes creates confusion (for example #105 ) minimum requirements for definition.h should be mentioned somewhere. I hav not yet able to build the project, so feel free add any changes

### CODE REVIEW

**Comment by mnumanuyar** [comment]
ok I just realised I grossly under-estimated how much is required from definitions.h

**Review by julianoes — APPROVED** [review]
Thanks, I think that's helpful.

File: README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ This repository contains user-space gps drivers, used as a submodule in
 All platform-specific stuff is done via a callback function and a
 `definitions.h` header file.
 
+In order for the project to build, definitions.h must include definitions for sensor_gnss_relative_s, sensor_gps_s and satellite_info_s.
```
Comment (bkueng) [inline README.md:10]:
```suggestion
In order for the project to build, `definitions.h` must include definitions for `sensor_gnss_relative_s`, `sensor_gps_s` and `satellite_info_s`.
```

File: README.md [hunk 2]
```diff
@@ -7,3 +7,5 @@ This repository contains user-space gps drivers, used as a submodule in
...
+For example implementation in [PX4 Firmware](https://github.com/PX4/PX4-Autopilot/blob/master/src/drivers/gps/definitions.h) or [QGroundControl](https://github.com/mavlink/qgroundcontrol/blob/master/src/GPS/definitions.h).
```
Comment (bkueng) [inline README.md:11]:
```suggestion
For example, check the implementation in [PX4 Autopilot](https://github.com/PX4/PX4-Autopilot/blob/master/src/drivers/gps/definitions.h) or [QGroundControl](https://github.com/mavlink/qgroundcontrol/blob/master/src/GPS/definitions.h). 
```

**Review by bkueng — APPROVED** [review]
Thanks

**Pull request merged**